app/models/qernel/node_api/cost.rb

```diff
@@ -222,6 +222,17 @@ def total_costs
         end
       end
 
+      # Internal: Calculates the total cost excluding fuel costs of a plant in euro per plant per year.
+      #
+      # Total cost is made up of OPEX and CAPEX (excluding fuel costs).
+      #
+      # Returns the total costs excluding fuel costs for one plant per year.
+      def total_costs_excluding_fuel_costs
+        fetch(:total_costs_excluding_fuel_costs) do
+          capital_expenditures + operating_expenses if capital_expenditures && operating_expenses
```

Comment:
I don't see `capital_expenditures` and `operating_expenses` defined anywhere in ETEngine. Furthermore, `total_costs` are built using `fixed_costs + variable_costs`, while `total_costs_excluding_fuel_costs` uses `capital_expenditures + operating_expenses`. I would then only expect the variable_costs component to be changed.

Reply:
@redekok Could it be that this was an aborted attempt to write a method, but that we ultimately opted for a manual, hard coded calculation in lines 213-214?